ext/random/tests/03_randomizer/user_unsafe.phpt

```diff
@@ -0,0 +1,83 @@
+--TEST--
+Random: Randomizer: User: Engine unsafe
+--FILE--
+<?php
+
+// Empty generator
+$randomizer = (new \Random\Randomizer(
+    new class () implements \Random\Engine {
+        public function generate(): string
+        {
+            return '';
+        }
+    }
+));
+
+try {
+    $randomizer->getInt(\PHP_INT_MIN, \PHP_INT_MAX);
+} catch (\RuntimeException $e) {
+    echo "{$e->getMessage()}\n";
+}
+
+try {
+    $randomizer->getBytes(1);
+} catch (\RuntimeException $e) {
+    echo "{$e->getMessage()}\n";
+}
+
+try {
+    $randomizer->shuffleArray(\range(1, 10));
+} catch (\RuntimeException $e) {
+    echo "{$e->getMessage()}\n";
+}
+
+try {
+    $randomizer->shuffleString('foobar');
+} catch (\RuntimeException $e) {
+    echo "{$e->getMessage()}\n";
+}
```

Comment:
```suggestion
try {
    var_dump($randomizer->getInt(\PHP_INT_MIN, \PHP_INT_MAX));
} catch (\RuntimeException $e) {
    echo "{$e->getMessage()}\n";
}

try {
    var_dump(bin2hex($randomizer->getBytes(1)));
} catch (\RuntimeException $e) {
    echo "{$e->getMessage()}\n";
}

try {
    var_dump($randomizer->shuffleArray(\range(1, 10)));
} catch (\RuntimeException $e) {
    echo "{$e->getMessage()}\n";
}

try {
    var_dump($randomizer->shuffleString('foobar'));
} catch (\RuntimeException $e) {
    echo "{$e->getMessage()}\n";
}
```

For consistency with the below, even if all of them throw. See parentheses in my previous review: https://github.com/php/php-src/pull/8094#discussion_r900478331